AISKU/rollup.config.js

```diff
@@ -19,28 +18,6 @@ const replaceValues = {
   "// Licensed under the MIT License.": ""
 };
 
-// Function to remove the @DynamicProtoStubs and rewrite the headers for the dist-esm files
```

Comment:
I believe the issue this problem was occuring for users directly consuming dist-esm modules files is because the previous change to change the copyright header (to include the version number) and the removal of the dynamicProto() stub functions in some cases caused some decoders of the map file throwing and reporting undefined into the output.

While I didn't repo the issue directly, I did find that by using this site to visual the map files showed a combination of an error for substr() and or additional "undefined" values being added to the end of the visualation.
http://sokra.github.io/source-map-visualization/#custom